EIPS/eip-fractional_gas_costs.md

```diff
@@ -0,0 +1,113 @@
+---
+eip: <to be assigned>
+title: Fractional gas costs for EVM opcodes
+author: Casey Detrio (@cdetrio)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category Core
+created: 2019-05-17
+---
+
+## Abstract
+According to recent benchmarks, EVM opcodes for computation (ADD, SUB, MUL, etc.) are generally overpriced relative to opcodes for storage I/O (SLOAD, SSTORE, etc.). Currently the minimum gas cost is 1 (i.e. one unit of gas), and most computational opcodes have a cost near to 1 (e.g. 3, 5, or 8), so the range in possible cost reduction is limited. A new minimum unit of gas, called a "particle", which is a fraction of 1 gas, would expand the range of gas costs and thus enable reductions below the current minimum.
```

Comment:
Can we see the benchmarks?  An appendix perhaps.  Also, the arithmetic opcodes are really algorithms, and take longer to run depending on the size of the inputs.  This is especially obvious with DIV and EXP.  So that's another reason the gas costs are wrong.

For instance, here are nanoseconds per DIV (aleth on a slow machine) with the operands restricted to <=64, <=128, and <=256 bits:
div64, 75
div128, 150
div256, 254

Reply:
threw up some benchmarks here for now: https://notes.ethereum.org/JNAmQfaSRvyUkdS42IaQMA?view